README.md

```diff
@@ -40,6 +46,7 @@ export LC_DOCKER_COMPOSE=/usr/local/bin/docker-compose
 export DOCKER_HOST=tcp://docker-daemon-host:2375
 
 ```
+Alternatively there is an installation script `bootstrap` to compile and install binaries
```

Comment:
This is nit-picky, but I would change this to "... installation script, `./bootstrap`, to compile...." Notice the commas.